### PR TITLE
chore(deps): google-cloud.version to 0.119.0-alpha

### LINKED ISSUE #2333: Upgrade google-cloud-bigtable 1.6.0 from 0.109.0: BulkMutationBatcher removed
When I tried to upgrade google-cloud-java version to 0.119 (https://github.com/googleapis/java-bigtable-hbase/pull/2332), the compilation failed for BulkMutationGCJClient.java because google-cloud-bigtable 1.6.0 does not have `com.google.cloud.bigtable.data.v2.models.BulkMutationBatcher` any more.

image

Can somebody fix this?

### PR DESCRIPTION
**Pull request by suztomo**
Fixes #2333

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,7 +59,7 @@ limitations under the License.
 
         <!-- google-cloud-java related dependency versions -->
         <!-- TODO: check if commons-codec was upgraded to 1.13 in org.apache.httpcomponents from http-client-->
-        <google-cloud.version>0.116.0-alpha</google-cloud.version>
+        <google-cloud.version>0.119.0-alpha</google-cloud.version>
```
Comment (suztomo) [inline pom.xml]: This is a version of google-cloud-**client** bom. The version 0.120.0-alpha does not have bigtable-related managed dependencies (See https://github.com/googleapis/java-bigtable-hbase/pull/2332).

**Comment by suztomo** [comment]
@kolea2 Would you review/merge this?

**Comment by suztomo** [comment]
Question: should bigtable-client-core continue to use google-cloud-**clients** or switch to google-cloud-**bom**?

As of now bigtable-client-core uses google-cloud-clients for dependency management ([bigtable-client-core/pom.xml](https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-client-core/pom.xml#L49)).

Google-cloud-client 0.119.0 has a little outdated dependency such as google-cloud-bigtable 1.6.0, which depends on grpc 1.24.0.

Google-cloud-clients's latest release (0.120.0) does not have google-cloud-bigtable. ([Here is the error](https://github.com/googleapis/java-bigtable-hbase/pull/2332#issue-357273943) when I tried to upgrade the version).

Google-cloud-bom's latest version 0.120.2 has google-cloud-bigtable 1.7.1, which depends on grpc 1.25.0.

**Comment by rahulKQL** [comment]
@suztomo  I also think we should use the latest google-cloud-bom(0.120.2), as we are already using the grpc-1.25.0 in for bigtable-client-core

https://github.com/googleapis/java-bigtable-hbase/blob/08ab65e7c4fc82a6b9015f9c2866fed1ed6d7b5e/pom.xml#L67

**Comment by suztomo** [comment]
@igorbernstein2 Feel free to close this PR if your change for the BOM can do the job.

**Comment by igorbernstein2** [comment]
closing in favor of #2345

**Pull request closed without merging**